kubernetes/gke-utility/argocd/clusters.yaml

```diff
@@ -23,3 +24,30 @@ stringData:
       }
     }
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gke-prow-build
+  labels:
+    argocd.argoproj.io/secret-type: cluster
+    clusterType: prow
+    environment: prod
+    prow-namespace: test-pods
```

Comment:
Should we also change that to `prowNamespace`?

Reply:
Oops, fixed